rails-auth.gemspec

```diff
@@ -29,6 +29,6 @@ Gem::Specification.new do |spec|
 
   spec.add_runtime_dependency "rack"
 
-  spec.add_development_dependency "bundler", "~> 1.10"
+  spec.add_development_dependency "bundler", [">= 1.10", "< 3"]
```

Comment:
nit: make this just `spec.add_development_dependency "bundler", ">= 1.10", "< 3"`